common/spring-cloud-stream-modules-common-configuration/src/test/java/org/springframework/cloud/stream/module/ftp/FtpSessionFactoryPropertiesTests.java

```diff
@@ -98,6 +99,7 @@ public void noPropertiesThrowsMeaningfulException() {
 			context.register(Conf.class);
 			context.refresh();
 			fail("should throw exception");
```

Comment:
From the catch block below, I guess it reads OK as it is now (although `ExpectedException` works too). Please disregard